packages/manager-components/tsconfig.json

```diff
@@ -7,14 +7,21 @@
     "outDir": "dist",
     "allowJs": false,
     "skipLibCheck": true,
-    "esModuleInterop": false,
+    "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "moduleResolution": "Node",
     "isolatedModules": true,
     "declaration": true,
     "declarationDir": "./dist/types",
-    "jsx": "react-jsx"
+    "jsx": "react"
   },
-  "include": ["src", "./src/**/*.ts", "./src/**/*.tsx", "*.spec.tsx"],
+  "files": ["global.d.ts"],
+  "include": [
```

Comment:
no need to list all file extensions, see : https://www.typescriptlang.org/tsconfig#include

src/**/* will handle both ts & tsx extensions

Reply:
👍